service/handler_combinedKey.go

```diff
@@ -0,0 +1,101 @@
+package service
```

Comment:
Pls rename this filename with `handler_combined_key.go` or `handler_secret_key.go`.